SeaPublicWebsite/Views/EnergyEfficiency/NewOrReturningUser.cshtml

```diff
@@ -65,16 +65,16 @@
 
     </div>
     <div class="govuk-grid-column-one-third">
-        <aside class="app-related-items" role="complementary">
-            <h2 class="govuk-heading-m" id="subsection-title">What does this mean?</h2>
+        <aside class="app-related-items" role="complementary" aria-labelled-by="helper text">

```

Comment:
I suspect this is meant to be `aria-label` instead of `aria-labelled-by`. The latter expects an element ID